src/app/hikes/hikes-list/hikes-list.component.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { StarComponent } from '../../shared/star/star.component';
+import { AppendMilesPipe } from '../../shared//append-miles.pipe';
```

Comment:
oops typo :L 

Also can we import the SharedModule into TestBed instead of doing them individually?

Reply:
Not sure what the typo is (?)

Also, you're right--I could import the whole module, especially since I'm using all two things out of it, but I thought we might show changing these 2 individual imports into importing the whole module as an example during the testing section.

If the module had 10 things, and we were only using 2, I would lean toward individual imports (principle of least privilege and all that).

It's definitely a good example of all the ceremony around testing in Angular.

Reply:
The typo: "../../shared//append-miles.pipe " It's got two slashes

As for the module for testing, okay, as long as we can make a note of that somewhere so we remember to do it, then that makes sense to me!

Reply:
I removed the extra slash, even though I really thought it pulled the room together.